LongDivision/src/ua/goit/alg/DivideNumbers.java

```diff
@@ -0,0 +1,88 @@
+package ua.goit.alg;
+
+import java.io.InputStream;
+import java.util.Scanner;
+
+public class DivideNumbers {
+
+    public static void main(String[] args) {
+        longDiv(System.in);
+    }
+
+    public static void longDiv(InputStream stream) {
+        System.out.println("Enter two integers through whitespace:");
+        Scanner sc = new Scanner(stream);
+        int dividend = sc.nextInt(); 
+        int first = dividend;
+        int divisor = sc.nextInt();
+        int second = divisor;
+        sc.close();
+
+        if (divisor == 0) {
+            throw new IllegalArgumentException("Error: Divide by zero!");
+        }
+
+        boolean isPoint = false;
+        StringBuilder divProcess = new StringBuilder();
+        StringBuilder result = new StringBuilder();
+        int accuracy = 5;
+        int decimal = 0;
+
+        if ((dividend < 0) ^ (divisor < 0)) {
```

Comment:
having xor is a bit unconventional :) please refactor a variable out for a documentation purpose